lib/metadataTypes/Query.js

```diff
@@ -452,6 +452,18 @@ class Query extends MetadataType {
         // delete local copy: retrieve/cred/bu/.../...-meta.sql
         await super.postDeleteTasks(customerKey, [`${this.definition.type}-meta.sql`]);
     }
+    /**
+     * Gets executed after deployment of metadata type
+     *
+     * @param metadata metadata mapped by their keyField
+     * @param {TYPE.MetadataTypeMap} _ originalMetadata to be updated (contains additioanl fields)
+     * @param {{created: number, updated: number}} createdUpdated counter representing successful creates/updates
+     */
+    static async postDeployTasks(metadata, _, createdUpdated) {
+        if (Util.OPTIONS.execute) {
+            this.execute(Object.keys(metadata));
+        }
```

Comment:
Execute both on deploy and update right? 

![image](https://github.com/Accenture/sfmc-devtools/assets/79999614/eba53537-33c3-42fc-9e56-0a6577c0100b)

Reply:
On create and update,  yes

Reply:
On create and update,  yes